client/src/Try/Gist.js

```diff
@@ -1,48 +1,9 @@
 "use strict";
 
-exports.getGistById_ = function(id, done, fail) {
-  $.ajax({
-    url: 'https://api.github.com/gists/' + id,
-    dataType: 'json'
-  }).done(done).fail(function(err) {
-    fail("Unable to load Gist metadata");
-  });
-}
-
-exports.tryLoadFileFromGist_ = function(gistInfo, filename, done, fail) {
+exports.rawUrl_ = function (gistInfo, filename) {
```

Comment:
This small function, `rawUrl_`, is copied from the beginning of `tryLoadFileFromGist_`:

```js
exports.tryLoadFileFromGist_ = function(gistInfo, filename, done, fail) {
  if (gistInfo.files && gistInfo.files.hasOwnProperty(filename)) {
    var url = gistInfo.files[filename].raw_url;
```

I opted to keep this code in the FFI rather than implement the decoding in PureScript, once again to keep the diff minimal. In the future, though, it would be nice to switch to something like `codec-argonaut` instead.